api/app.py

```diff
@@ -84,6 +85,9 @@ def create_flask_app_with_configs() -> Flask:
     dify_app = DifyApp(__name__)
     dify_app.config.from_mapping(dify_config.model_dump())
 
+    if dify_app.config.get("PROXY_FIX_MIDDLEWARE_ENABLED"):
```

Comment:
Based on @bowenliang123 's suggestion, I changed the implementation to an extension.

In other extensions, the `app.config.get()` is used to gather configuration, and I haven't seen any examples of `import` ing `dify_config` within the extensions.

Rather, this is an extension, I think it's better not to depend on `dify_config`. However, if you think it's better to use `dify_config`, please let me know, and I will revise it again.

Reply:
You can use `dify_config` confidently because even when using `app.config.get()`, the key used for retrieving values is still inherently dependent on dify_config.

Reply:
Agree, dify_config also guarantees the config value in proper data type rather than plain string.